pubspec.yaml

```diff
@@ -1,18 +1,17 @@
 name: shelf_web_socket
-version: 0.2.2+4
+version: 0.2.2+5
 
 description: A WebSocket handler for Shelf.
 author: Dart Team <misc@dartlang.org>
 homepage: http://github.com/dart-lang/shelf_web_socket
```

Comment:
while you're here: would you make this https?

Reply:
Sure